kubernetes/linux/setup.sh

```diff
@@ -9,92 +9,81 @@ else
     ARCH=$1
 fi
 
-#Download utf-8 encoding capability on the ama-logs container.
-#upgrade apt to latest version
-apt-get update && apt-get install -y apt && DEBIAN_FRONTEND=noninteractive apt-get install -y locales
+sudo tdnf install ca-certificates-microsoft -y
+sudo update-ca-trust
+
+# sudo tdnf install ruby-3.1.3 -y
+tdnf install -y gcc patch bzip2 openssl-devel libyaml-devel libffi-devel readline-devel zlib-devel gdbm-devel ncurses-devel
+wget https://github.com/rbenv/ruby-build/archive/refs/tags/v20230330.tar.gz -O ruby-build.tar.gz
+tar -xzf ruby-build.tar.gz
+PREFIX=/usr/local ./ruby-build-*/install.sh
+ruby-build 3.1.3 /usr
```

Comment:
please attach perf (cpu & memory) screenshots between the current released version vs mariner v2 upgrade.

Reply:
Updated this